src/corner-dialog/src/CornerDialog.js

```diff
@@ -233,7 +234,7 @@ export default class CornerDialog extends PureComponent {
               backgroundColor="white"
               elevation={4}
               width={width}
-              css={animationStyles}
```

Comment:
Seems like we could avoid this change if we pushed it into `Pane` and maybe `Text`. Anywhere that applies `css` to a `Box` component _directly_.

Reply:
Doesn't evergreen should follow ui-box way to drop support of css prop ?

Reply:
Yeah. I think we should mark it as deprecated so devs know to move off of it. Then in the next major version we can remove it completely.